Automattic/wp-calypso
# Atomic: Update eligibility checks

#### Changes proposed in this Pull Request

- Switch from site owner to site admins
- No more primary domain owner check
- Cleanup
- Add blocked transfer status

#### Testing instructions

- Create a simple site
- Make a non-admin user
- Shouldn't be able to transfer

-----------------------------------

- Make an admin user
- Should be able to transfer

-----------------------------------

- Add a domain with non admin user
- Try to go atomic with that domain as primary and admin user
- Should be able to transfer
